hack/00-osd-managed-cluster-config-integration.yaml.tmpl

```diff
@@ -26752,6 +26752,148 @@ objects:
+        operator: NotIn
+        values:
+        - 'true'
+      - key: api.openshift.com/addon-managed-api-service
+        operator: NotIn
+        values:
+        - 'true'
+    resourceApplyMode: Sync
+    resources:
+    - apiVersion: v1
+      kind: ServiceAccount
+      metadata:
+        name: sre-operator-reinstall-sa
+        namespace: openshift-observability-operator
+    - apiVersion: rbac.authorization.k8s.io/v1
+      kind: Role
+      metadata:
+        name: sre-operator-reinstall-role
+        namespace: openshift-observability-operator
+      rules:
+      - apiGroups:
+        - operators.coreos.com
+        resources:
+        - clusterserviceversions
+        - subscriptions
+        - installplans
+        - catalogsources
+        - operatorgroups
+        verbs:
+        - list
+        - get
+        - delete
+        - create
+      - apiGroups:
+        - batch
+        resources:
+        - cronjobs
+        verbs:
+        - list
+        - get
+        - delete
+      - apiGroups:
+        - ''
+        resources:
+        - serviceaccounts
+        verbs:
+        - list
+        - get
+        - delete
+      - apiGroups:
+        - rbac.authorization.k8s.io
+        resources:
+        - roles
+        - rolebindings
+        verbs:
+        - list
+        - get
+        - delete
+    - apiVersion: rbac.authorization.k8s.io/v1
+      kind: RoleBinding
+      metadata:
+        name: sre-operator-reinstall-rb
+        namespace: openshift-observability-operator
+      roleRef:
+        kind: Role
+        name: sre-operator-reinstall-role
+        apiGroup: rbac.authorization.k8s.io
+        namespace: openshift-observability-operator
+      subjects:
+      - kind: ServiceAccount
+        name: sre-operator-reinstall-sa
+        namespace: openshift-observability-operator
+    - apiVersion: batch/v1
+      kind: CronJob
+      metadata:
+        name: sre-obo-uninstall
+        namespace: openshift-observability-operator
+      spec:
+        ttlSecondsAfterFinished: 100
+        failedJobsHistoryLimit: 1
+        successfulJobsHistoryLimit: 3
+        concurrencyPolicy: Replace
+        schedule: '* * * * *'
+        jobTemplate:
+          spec:
+            template:
+              spec:
+                serviceAccountName: sre-operator-reinstall-sa
+                restartPolicy: Never
+                containers:
+                - name: operator-reinstaller
+                  image: image-registry.openshift-image-registry.svc:5000/openshift/cli:latest
+                  imagePullPolicy: Always
+                  command:
+                  - sh
+                  - -c
+                  - '#!/bin/bash
+
+                    set -euxo pipefail
+
+                    NAMESPACE=openshift-observability-operator
+
+                    oc -n "$NAMESPACE" get clusterserviceversions.operators.coreos.com
+                    | grep observability-operator | awk ''{print $1}'' | xargs oc
```

Comment:
This will conflict with the COO csv name:
```
$ oc -n openshift-observability-operator get csv
NAME                                       DISPLAY                          VERSION            REPLACES                                  PHASE
cluster-observability-operator.0.4.1       Cluster Observability Operator   0.4.1              cluster-observability-operator.0.3.2      Succeeded
observability-operator.v0.4.0              Observability Operator           0.4.0              observability-operator.v0.3.5             Failed
observability-operator.v0.4.1              Observability Operator           0.4.1              observability-operator.v0.4.0             Failed
```

The `grep` has to change to:

```
| grep "^observability-operator"
```